src/builder.rs

```diff
@@ -149,14 +158,17 @@ impl<K: EnrKey> EnrBuilder<K> {
 
         // Sanitize all data, ensuring all RLP data is correctly formatted.
         for (key, value) in &self.content {
-            if rlp::Rlp::new(value).data().is_err() {
+            if Bytes::decode(&mut value.to_vec().as_slice()).is_err() {
```

Comment:
```suggestion
            if Bytes::decode(&mut value.as_ref()).is_err() {
```
on the hunt for extra allocations we can remove